sky/provision/cudo/instance.py

```diff
@@ -77,10 +76,15 @@ def run_instances(region: str, cluster_name_on_cloud: str,
 
     created_instance_ids = []
     public_key = config.node_config['AuthorizedKey']
-
+    instance_type = config.node_config['InstanceType']
+    spec = cudo_machine_type.get_spec_from_instance(instance_type, region)
+    gpu_count = int(float(spec['gpu_count']))
+    vcpu_count = int(spec['vcpu_count'])
+    memory_gib = int(spec['mem_gb'])
+    gpu_model = spec['gpu_model']
+    cudo_wrapper.vm_available(to_start_count, gpu_count, gpu_model, region,
+                              memory_gib, vcpu_count)
```

Comment:
It would be nice to print out the error message to have a better UX : )
```suggestion
    try:
        cudo_wrapper.vm_available(to_start_count, gpu_count, gpu_model, region,
                              memory_gib, vcpu_count)
    except Exception as e:
        logger.warning('run_instances: {e}')
        raise
```